guides/novu-fcm-web.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: 'How to use FCM with Novu'
+description: 'Send FCM push notifications in a React app using Novu'
+icon: 'circle-6'
+---
+
+## Introduction
+In this guide, we'll learn how to send FCM push notifications in a web app using Novu. But before exploring the actual code, let’s understand what a push notification is and how it works.
+You can find the entire code (frontend as well as backend) for this app [here](https://github.com/novuhq/).
+<br/>
+Push notifications are notifications that are 'pushed' to a user's devices whether the user is using the app or not. We'll be using Firebase Cloud Messaging or FCM integration of Novu to send these notifications. This guide will be broken down into three parts:
```

Comment:
I think you can replace 'pushed' here with `sent`

Reply:
-> We'll be using Firebase Cloud Messaging or FCM integration of Novu to send these notifications

can be:

-> We'll be using Firebase Cloud Messaging (FCM) integration of Novu to send these notifications.